fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -4,7 +4,7 @@
   <record>
     hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
     message ${log}
-    version 1.0.6
+    version 1.1.4
```

Comment:
So this is just a marker for the data. Since we're making some bigger
changes like adding new data and changing the index formatting this was an
appropriate time to up this version.
On Mar 22, 2016 8:01 AM, "Jeff Cantrill" <notifications@github.com> wrote:

> In fluentd/configs.d/filter/k8s_record_transform.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/98#discussion_r56981288>
> :
>
> > @@ -4,7 +4,7 @@
> >    <record>
> >      hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
> >      message ${log}
> > -    version 1.0.6
> > +    version 1.1.4
>
> I see we are bumping these versions here. Are they needed for the PR or is
> it just an opportunity to upgrade?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/98/files/15769affe2fb22e7a8cdb1660ed0b2fe125879b5#r56981288>
>